linode/lke/cluster.go

```diff
@@ -128,22 +128,22 @@ func ReconcileLKENodePoolSpecs(
 	return result, nil
 }
 
-func waitForNodePoolReady(
+func WaitForNodePoolReady(
```

Comment:
It might make sense to move this helper over to the `nodepool` package since it's more directly related to pools than clusters

Reply:
done